apps/chromealive-extension/package.json

```diff
@@ -33,9 +36,13 @@
     "@vue/compiler-sfc": "^3.0.0",
     "autoprefixer": "^9",
     "babel-loader": "^8.2.2",
+    "chrome-api-mock": "^0.0.2",
     "copy-webpack-plugin": "^9.0.1",
     "glob": "^7.1.6",
+    "jest-chrome": "^0.7.2",
     "postcss": "^7",
+    "sass": "^1.43.4",
+    "sass-loader": "sass-loader@10.1.1",
```

Comment:
This seems to be breaking github actions

Reply:
Fixed

Reply:
Fixed